lib/transformer.js

```diff
@@ -434,7 +436,7 @@ Transformer.prototype.replace = function (rule, cb) {
   this.driver.grep(rule.search, function (err, grepResult) {
     var files = [];
 
-    // 1. Determine files & lines from grep
+    // 1. Determine files & lines from grep and filter out global excludes
```

Comment:
since you're not filtering here, doesn't make sense to include it in this comment